### PR TITLE
2160 manage rb users

### PR DESCRIPTION
**Pull request by nigel-lowry**
### Context

https://trello.com/c/6OMKc2II

### CODE REVIEW

**Comment by nigel-lowry** [comment]
[Content updates for Responsible Bodies inviting new users](https://trello.com/c/6OMKc2II/2160-content-updates-for-responsible-bodies-inviting-new-users)

**Review by markhallen — APPROVED** [review]

**Review by ralph-hawkins — APPROVED** [review]
💯  lovely

File: app/views/responsible_body/users/index.html.erb [hunk 1]
```diff
@@ -18,18 +18,13 @@
...
-    <p class="govuk-body">Everyone has the same level of access, they can:</p>
+    <p class="govuk-body">Everyone with access can:</p>
     <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-6">
-      <li>add or edit existing users</li>
-      <li>view the list of existing users</li>
-      <li>order devices for centrally managed schools and colleges</li>
-      <li>use a <%= govuk_link_to 'Support Portal', 'https://computacenterprod.service-now.com/dfe' %> to report faults and <%= govuk_link_to 'view passwords needed to replace DfE settings', devices_guidance_subpage_path('preparing-microsoft-windows-laptops-and-tablets', anchor: 'getting-local-admin-and-bios-passwords-to-install-your-own-software-and-settings') %> on Microsoft Windows devices</li>
+      <li>see who else has an account</li>
+      <li>add or edit existing accounts</li>
+      <li>access the support portal</li>
```
Comment (Lloyd-K) [inline app/views/responsible_body/users/index.html.erb:25]: Please capitalise 'Support Portal'

Comment (nigel-lowry) [reply app/views/responsible_body/users/index.html.erb:25]: Fixed now on the service closure branch

**Pull request merged**